src/tests/lvm/lvm_thick_volume_resize/util.go

```diff
@@ -0,0 +1,117 @@
+package lvm_thick_volume_resize
```

Comment:
Should this file 'util.go' exist in a path outside the "lvm_thick_volume_resize" folder? It seems to be used for both thick as well as thin tests.

Reply:
Yes, it make sense to keep this file in common folder outside any thick/thin test and use it from there.
I've moved it here -- https://github.com/openebs/openebs-e2e/pull/42/commits/b1287dbe14a28220bd825d7b8018f4bf60fcabf8